apps/condo/domains/user/gql.js

```diff
@@ -192,6 +190,15 @@ const CHECK_USER_EXISTENCE_MUTATION = gql`
 const RESET_USER_LIMIT_ACTION_FIELDS = `{ type identifier reason ${COMMON_FIELDS} }`
 const ResetUserLimitAction = generateGqlQueries('ResetUserLimitAction', RESET_USER_LIMIT_ACTION_FIELDS)
 
+const USER_SUDO_TOKEN_FIELDS = `{ token expiresAt user { id } remainingUses ${COMMON_FIELDS} }`
```

Comment:
I'd have renamed remainingUses to expiresAfter, just to keep naming in one style , but this is not a big deal

Reply:
I would leave it as is - to keep the API consistent, in other schemes we use the same fields for expiration time.